src/HW/LedgerConnect/DeviceItem/DeviceItem.tsx

```diff
@@ -7,9 +7,10 @@ import {Device} from '../../../yoroi-wallets/types'
 type Props = {
   device: Device
   onSelect: (device: Device) => Promise<void> | void
+  disabled?: boolean
 }
 
-export const DeviceItem = ({device, onSelect}: Props) => {
+export const DeviceItem = ({device, onSelect, disabled}: Props) => {
```

Comment:
Add the option to mark `DeviceItem` as disabled

Reply:
I've also added `opacity: 0.5` to the button when it's disabled, as it was not clear if the button is disabled or not. Please let me know your thoughts.